examples/django/oauth_app/slack_datastores.py

```diff
@@ -33,9 +33,9 @@ def save(self, installation: Installation):
         i = installation.to_dict()
         if is_naive(i["installed_at"]):
             i["installed_at"] = make_aware(i["installed_at"])
-        if "bot_token_expires_at" in i and is_naive(i["bot_token_expires_at"]):
+        if i.get("bot_token_expires_at") and is_naive(i["bot_token_expires_at"]):
```

Comment:
Can you change this way for consistency?
```suggestion
        if i.get("bot_token_expires_at") is not None and is_naive(i["bot_token_expires_at"]):
```